### DISCUSSION TITLE
Set cancellation status for order

### DISCUSSION THREAD

**Discussion by CMeldgaard** [craftcms/commerce#2407] [Ideas]
**What are you trying to do?**
Im trying to define at what status a order is cancelled. I can't delete a cancelled order, because i have external integrations that needs to be able to fetch orderdata, and herunder the order status. I know i can create my own statuses, but those orders will still be calculated in the different dashboard widget etc.

**What's your proposed solution?**
I proprose that, just as we can select an status to be the default order status, we should be able to mark if an order is cancelled / should be excluded on the order status.

**Comment by lukeholder** [comment]
Thanks for the idea. Will keep this enhancement suggestion open for further input from others.
